lib/cli/node-modules-path.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path  = require('path');
+
+module.exports = function(context) {
+  var nodePath = process.env.EMBER_NODE_PATH;
```

Comment:
@steffanpenner NODE_PATH brings extra baggage with it. It will try to load globally installed modules and node_modules installed in $HOME which we might want to avoid. My goal here is t be able to say I have a clean controlled node_modules path but it is not necessarily at the project root.